apps/inbound-mail/src/server/inbound-mail-queue.service.ts

```diff
@@ -1,9 +1,10 @@
 import { BullMqService } from '@novu/application-generic';
+import { QueueNameEnum } from '@novu/shared';
 import { QueueBaseOptions } from 'bullmq';
 
-export class QueueService {
+export class InboundMailQueueService {
```

Comment:
Renaming to avoid potential clashings in auto-magic Nest.js dependency injection. If it ever happens what we are guessing.